### PR TITLE
refac: grid loss responsible list

### PR DESCRIPTION
**Pull request by johevemi**

### CODE REVIEW

File: source/databricks/calculation_engine/package/calculation/preparation/transformations/grid_loss_responsible.py [hunk 1]
```diff
@@ -58,112 +58,112 @@ def _utc(date: str | None) -> datetime.datetime | None:
...
-    ("2834522000000000",),
-    ("1207543000000000",),
-    ("2071017000000000",),
-    ("2119017000000000",),
-    ("2920882000000000",),
-    ("3318924000000000",),
-    ("1581021000000000",),
-    ("3678454000000000",),
-    ("1797183000000000",),
+    ("742819000000000000",),
```
Comment (BjarkeMeier) [inline source/databricks/calculation_engine/package/calculation/preparation/transformations/grid_loss_responsible.py:61]: We have different input data sets in test_001. Which is these for? Should the comment be updated? Should the list be split per data set?

Comment (johevemi) [reply source/databricks/calculation_engine/package/calculation/preparation/transformations/grid_loss_responsible.py:61]: it is an update of the data we are still using, the metering_point_id in test001 performance tables has been updated to the correct char length and this is why this needs to be updated. We are not using the grid_loss_responsible from delta table yet.

Comment (johevemi) [reply source/databricks/calculation_engine/package/calculation/preparation/transformations/grid_loss_responsible.py:61]: I might not understand your question

Comment (johevemi) [reply source/databricks/calculation_engine/package/calculation/preparation/transformations/grid_loss_responsible.py:61]: I commented in the code which is for performance test and which is for manual test (not sure that is the best word, so if you have any suggestions)

**Review by BjarkeMeier — APPROVED** [review]

**Review by BjarkeMeier — APPROVED** [review]

**Pull request merged**